Library/Homebrew/dev-cmd/bump-cask-pr.rb

```diff
@@ -89,6 +89,14 @@ def bump_cask_pr
     odie "This cask is not in a tap!" if cask.tap.blank?
     odie "This cask's tap is not a Git repository!" unless cask.tap.git?
 
+    autobump_file_path = cask.tap.path/".github/autobump.txt"
+    if cask.tap.core_cask_tap? && File.readlines(autobump_file_path).grep(/#{cask.name}/).any?
```

Comment:
```suggestion
    if cask.tap.core_cask_tap? && File.readlines(autobump_file_path, chomp: true).include?(cask.name)
```

This way we can avoid the regex entirely.

Reply:
```suggestion
    if cask.tap.core_cask_tap? && autobump_file_path.exist? && autobump_file_path).readlines(chomp: true).include?(cask.name)
```

for using `Pathname` methods